### ISSUE TITLE
Android: Allow manually editing fee rates

### ISSUE THREAD

**Issue by gits7r** [spesmilo/electrum#6710]
On Android (Kivy) it is not possible to edit fees manually.  It would be nice to have a simple text box above the slider with sat / byte description.

Currently on Kivy you can only use the slider and see what fee rate it pre-filled for you, you cannot manually edit the fee rate manually for any of the fee methods available (Static, ETA, Mempool).

#5507 related to Android / Kivy fees.

**Comment by SomberNight** [comment]
Hmm I would have thought there was already an issue for this but guess not :)
Perfectly reasonable request for the custom fees.

**Comment by gits7r** [comment]
Yes, and one more thing, since it's related to custom fees it's not worth to open another issue about it but it makes sense to allow the user to set a fee of more than 300 sat  / byte. Currently this is the hard coded maximum limit. It's not enough, I currently have some transactions unconfirmed for several days, because of mempool backlog (some parents of parents have ultra low fee) and in order to get them confirmed I have to do a child transaction with way more than 300 sat  / byte.

It makes sense for beginners to have a "soft limit" of some kind, but even this should be something dynamic based on server average / mempool something, but power users should have the option to override the fee and set as much as they want, even if they will get a warning from Electrum.  Otherwise we just have to "tweak" the hard coded max limit from time to time when network conditions change which is not great.

**Comment by Sainan** [comment]
That fee rate slider is probably the number 1 reason you are losing users. ;D

**Comment by SomberNight** [comment]
note: the kivy GUI is no longer maintained -- but its replacement, the qml GUI, is also missing this

**Comment by RoMi1981** [comment]
This request is open since 5 years, crazy... Is there a plan to implement this? 

Thank you

**Comment by accumulator** [comment]
> This request is open since 5 years, crazy... Is there a plan to implement this?
> 
> Thank you

Yes. Soon™
